Sources/Features/Shared/CloudKit/CloudKitOperationExtensions.swift

```diff
@@ -19,6 +19,42 @@ extension OPRCKOperation where T: CKOperationType {
         get { return operation.container }
         set { operation.container = newValue }
     }
+
+    var allowsCellularAccess: Bool {
+        get { return operation.allowsCellularAccess }
+        set { operation.allowsCellularAccess = newValue }
+    }
+
+    @available(iOS 9.3, tvOS 9.3, OSX 10.12, watchOS 2.3, *)
+    var operationID: String {
+        get { return operation.operationID }
+    }
+
+    @available(iOS 9.3, tvOS 9.3, OSX 10.12, watchOS 2.3, *)
+    var longLived: Bool {
+        get { return operation.longLived }
+        set { operation.longLived = newValue }
+    }
+
+    #if swift(>=3.0) // TEMPORARY FIX: Swift 2.3 compiler crash (see: CloudKitInterface.swift)
+        @available(iOS 9.3, tvOS 9.3, OSX 10.12, watchOS 2.3, *)
+        var longLivedOperationWasPersistedBlock: () -> Swift.Void {
```

Comment:
A leftover from matching how they were defined in Swift 3. Can change it to Void for Swift 2.3.